libraries/app/include/graphene/app/application.hpp

```diff
@@ -40,6 +40,7 @@ namespace graphene { namespace app {
       public:
          bool enable_subscribe_to_all = false;
          bool has_market_history_plugin = false;
+         uint32_t max_account_history_operations_limit=100;
```

Comment:
I think this is not needed.

Reply:
-Not needed for base scenario 
-However needed for scenario from test case where initialization is not provided. Additionally this is part of the clarification I needed relating to below. More details in the main ticket (#782)

Needed clarification on :
1.
On running function get_account_history_operations located in file history_api_test.cpp, config.init file is not loaded , can you suggest any approach to test the API ?
image

I am using ilog function for debugging the shared_pointers as I face BOOST error for the RTTI symbols not found. In my company the team uses CLION IDE with GDB and found it to be more time-saving than ilog. I am open for any better way to debug and open to connect with any technical member for the same.

Reply:
> On running function get_account_history_operations located in file history_api_test.cpp, config.init file is not loaded , can you suggest any approach to test the API ?

To my knowledge, we don't have tests that use config files. Best approach might be to derive a subclass from database_fixture that creates and uses a config file in `graphene::utilities::temp_directory_path()`.

Reply:
> > On running function get_account_history_operations located in file history_api_test.cpp, config.init file is not loaded , can you suggest any approach to test the API ?
> 
> To my knowledge, we don't have tests that use config files. Best approach might be to derive a subclass from database_fixture that creates and uses a config file in `graphene::utilities::temp_directory_path()`.

Thank you I shall explore above approach.
In addition if I start witness node and hit the API for function get_account_history_operations, would this option work? I shall explore this too and update.